README.md

```diff
@@ -84,11 +83,11 @@ client.login("token");
 - [GitHub](https://github.com/OpenianDevelopment/chiro.js)
 - [NPM](https://www.npmjs.com/package/chiro.js)
 
-
 ## Help
 
 If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [Chiro.js Server](https://menhera-chan.in/support).
+nudge in the right direction kindly create an github issue or join our official [Chiro.js Server](https://menhera-chan.in/support).
 
 ## Notice
-The documentation is WIP. This is just a temporary docs
+
+The documentation is WIP. This is just a temporary docs
```

Comment:
Docs are generated with GitHub CI and are rendered here https://docs.snowflakedev.org/docs/chiro/main/general/welcome

Reply:
Actually it wasn't me. It was rohnak who wrote there though.

Reply:
And he has already a link https://chirojs.openian.dev/